lib/components/home/move_note_button.dart

```diff
@@ -38,49 +36,46 @@ class _MoveNoteDialog extends StatefulWidget {
   const _MoveNoteDialog({
     // ignore: unused_element
     super.key,
-    required this.existingPath,
+    required this.filesToMove,
   });
 
-  final String existingPath;
+  final List<String> filesToMove;
 
   @override
   State<_MoveNoteDialog> createState() => _MoveNoteDialogState();
 }
 class _MoveNoteDialogState extends State<_MoveNoteDialog> {
-  /// The original file name of the note.
-  late String fileName = widget.existingPath.substring(
-    widget.existingPath.lastIndexOf('/') + 1,
-  );
-  /// The original parent folder of the note,
+  /// The original file names of the notes.
+  late List<String> fileNames = widget.filesToMove.map((path) => path.substring(path.lastIndexOf('/') + 1)).toList();
+
+  /// The original parent folder of the notes,
   /// including the trailing slash.
-  late String parentFolder = widget.existingPath.substring(
-    0,
-    widget.existingPath.lastIndexOf('/') + 1,
-  );
+  
+  late String parentFolder = widget.filesToMove[0].substring(0, widget.filesToMove[0].lastIndexOf('/') + 1);
```

Comment:
I don't think the `filesToMove` always have the same parent folder. Maybe this would be better as a `List<String>` like `fileNames`?